docs/src/get-started/deploy/commit.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Git commit
+weight: -9
+description: Start getting your project into Platform.sh.
+---
+
+Once you have your project initialized, it's time to add the basics to get it deployed.
+
+Create a file to hold your app configuration:
+
+```bash
+touch .platform.app.yaml
+```
+
+This file holds all the configuration for the container where your app lives.
+By keeping the configuration in this file,
+you ensure it remains the same across all environments, from development to production.
+
+Start by adding basic properties for your app, such as its name and language.
+With all examples, you can adjust file to fit your code, such as the `start` command, or desires, such as changing the `name`.
```

Comment:
```suggestion
With all examples, you can adjust the file to fit your needs, such as the `start` command, or desires, such as changing the `name`.
```

Reply:
The idea with `code` was that it's really about fitting the existing code, not just needs for features or whatever. The missing `the` is definitely a good catch, though.